web/src/Styles/BreadCrumbs/BreadCrumbs.scss

```
@@ -0,0 +1,42 @@
+@import "../variables";
+
+#desktop {
+  @include respond-to("small") {
+    display: none;
+  }
+  @include respond-to("medium") {
+    display: block;
+  }
+  @include respond-to("large") {
+    display: block;
+    margin-left: 15%;
```

I believe `display: block` is the default style for most elements thus if unspecified, that style would not be applied thus no code is required to handle it.
```suggestion
  @include respond-to("large") {
    margin-left: 15%;
```

---

That's a good point thank you, we will take that out and double check it still works

---

worked like a charm thank you